pandas/core/tools/datetimes.py

```diff
@@ -759,13 +766,15 @@ def to_datetime(
         <https://docs.python.org/3/library/datetime.html
         #strftime-and-strptime-behavior>`_ for more information on choices, though
         note that :const:`"%f"` will parse all the way up to nanoseconds.
+        You can also pass "ISO8601" to parse any ISO8601 time string.
```

Comment:
Add a link that explains ISO8601? (eg https://en.wikipedia.org/wiki/ISO_8601)

Although looking at that link makes me wonder if we then should be explicit about that this is only the calendar date + time formatting (and not week dates or ordinal dates)